app/apps/server/bridges/scheduler.ts

```diff
@@ -1,19 +1,21 @@
-import Agenda from 'agenda';
+import { Agenda, Job } from 'agenda';
 import { ObjectID } from 'bson';
 import { MongoInternals } from 'meteor/mongo';
 import { StartupType, IProcessor, IOnetimeSchedule, IRecurringSchedule } from '@rocket.chat/apps-engine/definition/scheduler';
 import { SchedulerBridge } from '@rocket.chat/apps-engine/server/bridges/SchedulerBridge';
 
 import { AppServerOrchestrator } from '../orchestrator';
 
-function _callProcessor(processor: Function): (job: Agenda.Job) => void {
+const { MONGO_URL = 'mongodb://localhost:3001/meteor' } = process.env;
```

Comment:
Is the mongo URL actually on port 27017? Or is that /meteor telling it something else?

Reply:
You are right. On my local it was `3001/meteor` but I should be 27017. I'm updating it. Thanks!